proposals/mail-helper-refactor.md

```diff
@@ -105,7 +105,7 @@ import sendgrid
 from sendgrid.helpers.mail import *
 
 msg = Mail(from_email=From('from@example.com', 'From Name'),
-           to_email=To('to@example.com', 'To Name'),
+           to_emails=To('to@example.com', 'To Name'),
```

Comment:
This is a single email address, should it stay singular?

Reply:
I believe it should stay singular in this case.